components/utils.py

```diff
@@ -1158,7 +1181,18 @@ def _load_node_props(service_name):
 
     def _get_local_file_path(self, service_name, resource_name):
         base_service_res_dir = self.get_resources_dir(service_name)
-        dest_file_path = os.path.join(base_service_res_dir, resource_name)
+
+        if (resource_name.startswith('cloudify')
```

Comment:
please move this complicated condition into a function with explanatory name and comments..